src/dashboard-refactor/types.tsx

```diff
@@ -12,6 +12,11 @@ export interface DropReceivingState {
     onDrop(normalizedPageUrl: string): void
 }
 
+export interface TextPart {
+    text: string
+    match: boolean
```

Comment:
I'd either rename the type to something `SearchResultText` to highlight in what context it's meant to be used, or rename the `match` property to  `bold` so that instead it becomes a purely stylistic thing.